seedpod_ground_risk/core/plot_server.py

```diff
@@ -1,3 +1,4 @@
+import os
```

Comment:
Please use lazy importing for external dependencies, as the rest of the class does not depend on `os`